### ISSUE TITLE
Bug: Groupmode - message to teachers do only contain hard coded "groupname" string

### ISSUE THREAD

**Issue by Kathrin84** [academic-moodle-cooperation/moodle-mod_organizer#41]
Hi there,

a user of our instance reported that there's a bug in the message he receives from your plugin, because the groupname of the related group is not filled and instead he only gets a placeholder string (groupname):

(Did receive the message in German)
```
Im Rahmen des Kurses [], hat Teilnehmer/in Eva [] die Gruppe groupname für den Zeitslot am 3.08.2018 um 10:40 im/in [] angemeldet.
```

This is set here:
https://github.com/academic-moodle-cooperation/moodle-mod_organizer/blob/master/messaging.php#L115

Because the function "organizer_fetch_user_group()" would only return a group, if the teacher would be a member of the related group himself:
https://github.com/academic-moodle-cooperation/moodle-mod_organizer/blob/master/locallib.php#L1175

But usually the teacher is _not_ a member in all the groups in his course. So he will always receive the messages with the placeholder and therefore important information is lost for him.
I think I am quite sure that this already worked correctly - so there must be a change in the more recent time.

Best, Kathrin

**Comment by mkemmerling** [comment]
Hi Kathrin,

thanks for reporting. According to Thomas this bug has already been fixed in the upcoming version for Moodle 3.5. A little bit of patience please ...

Markus

**Comment by Kathrin84** [comment]
Hi Markus,
thank you for your answer and happy to hear that you already addressed and fixed this bug. Looking forward to the new release.
Best, Kathrin
